packages/hoppscotch-common/src/platform/index.ts

```diff
@@ -52,6 +52,12 @@ export type PlatformDef = {
      * Whether to show the A/B testing workspace switcher click login flow or not
      */
     workspaceSwitcherLogin?: Ref<boolean>
+
+    /**
+     * There's an active issue wrt syncing in personal workspace under SH while duplicating a collection
+     * This is a temporary flag to disable the same
+     */
+    duplicateCollectionDisabledInPersonalWorkspace?: boolean
```

Comment:
This is a temporary feature flag and the naming convention is slightly different `propEnabled` to prevent the need for updates at the `central` platform level config.